pydra/utils/typing.py

```diff
@@ -55,22 +56,33 @@ class TypeParser(ty.Generic[T]):
         the tree of more complex nested container types. Overrides 'coercible' to enable
         you to carve out exceptions, such as TypeParser(list, coercible=[(ty.Iterable, list)],
         not_coercible=[(str, list)])
+    label : str
+        the label to be used to identify the type parser in error messages. Especially
+        useful when TypeParser is used as a converter in attrs.fields
     """
 
     tp: ty.Type[T]
     coercible: ty.List[ty.Tuple[TypeOrAny, TypeOrAny]]
     not_coercible: ty.List[ty.Tuple[TypeOrAny, TypeOrAny]]
+    label: str
 
     COERCIBLE_DEFAULT: ty.Tuple[ty.Tuple[type, type], ...] = (
-        (ty.Sequence, ty.Sequence),  # type: ignore
-        (ty.Mapping, ty.Mapping),
-        (Path, os.PathLike),
-        (str, os.PathLike),
-        (os.PathLike, Path),
-        (os.PathLike, str),
-        (ty.Any, MultiInputObj),
-        (int, float),
+        (
+            (ty.Sequence, ty.Sequence),  # type: ignore
```

Comment:
What are you ignoring here? Often clarifying to use `ignore[XXX]`.

Reply:
That is a very good question. I just removed the ignore statement and it isn't showing any issues, so it must have been a hangover from some point.

Reply:
I have removed it now